client/src/components/retrieveQuizData.js

```diff
@@ -1,16 +1,3 @@
-import htmlQuiz from '../data/dummy.json';
-import cssQuiz from '../data/cssQuiz.json';
-import jsQuiz from '../data/jsQuiz.json';
-import { shuffle } from '../helpers/shuffle';
-
-const data = { html: htmlQuiz, css: cssQuiz, js: jsQuiz };
-
-export const retrieveQuizData = quizName => {
-  if (data[quizName]) {
-    const quizData = data[quizName];
-    quizData.forEach(question => shuffle(question.answers));
-    shuffle(quizData);
-    return quizData;
-  }
-  return [];
+export const getData = quizName => {
+  return fetch(`/quizzes?quizName=${quizName}`).then(res => res.json());
```

Comment:
```suggestion
export const getData = quizName =>
  fetch(`/quizzes?quizName=${quizName}`).then(res => res.json());
```
If a function body has only one line which is a return, then we can directly return this one line and remove the braces and `return`

Reply:
I see, thanks.